docs/content/docs/blog/release-osp-1.5.md

```diff
@@ -0,0 +1,96 @@
+---
+title: OpenShift Pipeline 1.5 PAC release
+---
+
+OpenShift Pipeline 1.5 introduces many new features for Pipelines-as-Code, here
+is a list of the most important ones.
+
+## GitOPS Commands
+
+The GitOPS commands are those small comments you can do a on pull request to
+trigger some actions on your pipelinerun via Pipelins-as-Code.
+
+The one you probably already know is `/test pipelinerun` to retest a specific
+pipelinerun or `/retest` to retest all the PipelineRuns.
+
+### Explicitly trigger any PipelineRuns without matching annotations
+
+It was previously only possible to trigger those PipelineRuns if they were
+matching the annotations. For examples the PipelineRuns would need to have the
+annotation `pipelinesascode.tekton.dev/on-event` set to `pull_request` (or the
+CEL equivalent) to be able to be re-triggered with `/test`.
+
+That limitation has been removed and now you can trigger any PipelineRuns with
+`/test` even if it doesn't match. This is useful for example if you don't want to
+have a long running pipelinerun running and consuming resources on every
+iteration of a pull request but you still want to run it explicitly just before
+merge the PR.
+
+### Redefine parameters on the fly on GitOPS commands
+
+When you run a PipelineRun you have a number of standard parameters that are
+automatically exposed to the PipelineRun, like `{{ revision }}` will be
+expanded to the revision from where the event come from.
+
+The GitOPS commands now accepts arguments of the form of key=value that will
+redefine those parameters including the custom params. So for example you can do:
+
+```shell
+/test pipelinerun revision=main
+```
+
+It will test the pipelinerun on the main branch instead of the commit of your
+Pull Request. The only parameters that can redefine are the standard ones and
+the extra ones you add as custom params on your `Repository` CR.
+
+### Define your own GitOPS commands
+
+In the Pipelines-as-Code version shipped with OpenShift Pipelines 1.5 we have
+introduced a new on event annotations, called
+`pipelinesascode.tekton.dev/on-comment` This let you start a PipelineRun when
+the comment has matched the regex specified in the annotation.
+
+This let you for example define your own GitOPS command and start a PipelineRun
+on everything that matches the regexp
+
+To make it even more powerful we have add a new standard parameter called `{{
+trigger_comment }}` which would be available to the PipelineRun and would
+contain the whole comment that triggered the PipelineRun.
+
+### Errors are now cleared between retest runs
```

Comment:
is this talking about failure checkruns ???

Reply:
yes